app/assets/javascripts/application.js

```diff
@@ -36,12 +36,6 @@
 //= require _shared
 //= require navbar
 
-// Components
-//= require components
-
 // Controller-specific
 //= require faculty
-//= require founders
-//= require story
 //= require resources
-//= require intercom_settings
```

Comment:
`/coaches` page is breaking. 
```
couldn't find file 'inspectlet.source.coffee'
```
Trace:  `app/views/layouts/application.html.erb:17`